ibis/tests/expr/test_table.py

```diff
@@ -2077,3 +2077,87 @@ def test_unbind_with_namespace():
 
     assert s.op() == expected.op()
     assert s.equals(expected)
+
+
+def test_table_bind():
+    def eq(left, right):
+        return all(a.equals(b) for a, b in zip(left, right))
+
+    t = ibis.table({"a": "int", "b": "string"}, name="t")
+
+    # boolean literals
+    exprs = t.bind(True, False)
+    expected = (ibis.literal(True), ibis.literal(False))
+    assert eq(exprs, expected)
+
+    # int literals
+    exprs = t.bind(1, 2)
```

Comment:
can we get a test for when the resulting columns would have duplicate names? eg `t.bind(1, 1)`

Reply:
I see what you're saying, but I think I still want it. Perhaps in the future we switch to a different implementation, would still want the test.

Reply:
Let's separate blocking comments from non blocking comments.

Adding a test for this should in no way block this PR from landing.

I disagree that we should add this test. It adds no material value to testing the behavior of `bind`. Adding a test that exercises duplicate column detection is just wasted CPU cycles and maintainer effort.

Reply:
So bind's responsibility to coerce the input arguments to a series of value expressions, nothing more.

Other methods like `.select() / .aggregate() / ...` do ensure that there are no duplicate names by calling `unwrap_aliases()`, so the check does happen but at a different level.

Reply:
OK, I think this is a misunderstanding caused by me not explaining what I thought the expected behavior should be. I have always used .bind in the larger context of `t.mutate(t.bind(x))`, so I have never wanted duplicate column names, so I originally thought that `t.bind(1, 1)` should error. But

> So bind's responsibility to coerce the input arguments to a series of value expressions, nothing more.

makes sense. I think you're right, dupe results should be allowed, and treated the same as any other result, and thus don't need a test. I was operating under the assumption they should be treated differently, and thus should have a test.

Reply:
I would maybe tweak the phrasing of the responsibility to "coerce the arguments to a tuple of value expressions (if they aren't already)." to emphasize that we don't do anything more clever/furhter inspection if the arg is already a value expression. This makes it more clear that `t.bind(other_table.column)` just results in `other_table.column`. I think this would be a good candidate to go in the summary line of the docstring.